src/constants.lisp

```diff
@@ -0,0 +1,15 @@
+;;;; constants.lisp
+;;;;
+;;;; This file contains constant values used throughout compilation.
+
+(defpackage #:coalton-impl/constants
+  (:use #:cl)
+  (:export
+   #:+default-loop-label+               ; VARIABLE
+   ))
+
+(in-package #:coalton-impl/constants)
+
+(defparameter +default-loop-label+ :coalton-loop
```

Comment:
`defconstant`

Reply:
Ah yeah I never use defconstant b/c I basically only hack on things with interactive development in mind and defconstant is too much of a pain.